ChemistryLib/PhreeqcInterfaceData/EquilibriumPhase.h

```diff
@@ -0,0 +1,46 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <boost/optional/optional.hpp>
+#include <fstream>
```

Comment:
Same here with the iosfwd. I thought you have already fixed these changes and I remember to have "resolved" that issue...

Reply:
It is my fault. The commit was reverted. It worked fine on my personal computer and also on Haibing's laptop with the header file <iosfwd> instead of <fstream>, while the automated check on Jenkins and Azure returns compilation failure. The error message is that no match for 'operator<<' (operand types are 'std::ofstream {aka std::basic_ofstream<char>}' and 'const string {aka const std::__cxx11::basic_string<char>}').

Reply:
Can you point to the log, please?

Reply:
Output.h still has an fstream include.

Reply:
Replaced with `<iosfwd>` and added `<fstream>` in `Output.cpp`.